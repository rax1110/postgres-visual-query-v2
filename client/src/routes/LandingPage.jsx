import React from 'react';
import { Col, Container, Row } from 'reactstrap';
import LoginFormContainer from '../components/LoginFormContainer';
import LanguageSwitcher from '../components/LanguageSwitcher';
import InformationPopover from '../components/InformationPopover';

const LandingPage = () => (
  <Container className="vh-100 pt-4">
    <Row>
      <Col>
        <LanguageSwitcher />
      </Col>
      <InformationPopover />
    </Row>
    <Row className="justify-content-center pt-4">
      <Col sm="9" md="7" lg="5" className="">
        <LoginFormContainer />
      </Col>
    </Row>
  </Container>
);

export default LandingPage;
