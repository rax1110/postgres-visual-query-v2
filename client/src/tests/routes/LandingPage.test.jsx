import React from 'react';
import { shallow } from 'enzyme';
import LandingPage from '../../routes/LandingPage';

describe('Route: LandingPage', () => {
  let component;

  test('LandingPage renders', () => {
    component = shallow(<LandingPage />);

    expect(component).toMatchSnapshot();
  });
});
