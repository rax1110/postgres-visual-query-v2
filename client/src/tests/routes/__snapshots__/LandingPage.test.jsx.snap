// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Route: LandingPage LandingPage renders 1`] = `ShallowWrapper {}`;
